solr/CHANGES.txt

```diff
@@ -48,6 +48,8 @@ Bug Fixes
 
 * SOLR-16075: ShowFileHandler path parameter is now validated to be relative to instance conf dir in standalone mode (janhoy)
 
+* SOLR-16143: SolrConfig ResourceProvider can miss updates from ZooKeeper (Mike Drob)
```

Comment:
As a general comment of CHANGES.txt entries, I'd prefer we word them in language we can all understand.  I didn't know of this "ResourceProvider"; it's very internal.  Instead, maybe try to say briefly what can go wrong?

Reply:
Updated this to be a little more natural language, will need to go back and update the 9x lines too to reflect the new verbiage once this is finalized.